### PR TITLE
wip Feat/msaas

### PR DESCRIPTION
**Pull request by BelfordZ**
addresses #20

### CODE REVIEW

File: service-mode-openrpc.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  "servers": [
+    {
+      "name": "production",
+      "url": "https://mock.open-rpc.org"
+    }
+  ],
+  "methods": [
+    {
+      "name": "mock",
+      "description": "Provides a mock server for the given OpenRPC document.",
```
Comment (shanejonas) [inline service-mode-openrpc.json]:
```suggestion
      "description": "Provides a mock server for the given OpenRPC document. This mock lasts 15 minutes before being destroyed.",
```

File: service-mode-openrpc.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "license": {
+      "name": "Apache 2.0",
+      "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "servers": [
+    {
+      "name": "production",
+      "url": "https://mock.open-rpc.org"
+    }
```
Comment (shanejonas) [inline service-mode-openrpc.json]: we should add `development`, pointing to `localhost:3333`

**Comment by zcstarr** [comment]
@BelfordZ  What's the intent of this ? Just so I can have it in my mind :D

**Pull request merged**